becquerel/core/spectrum.py

```diff
@@ -973,12 +973,15 @@ def find_bin_index(self, x, use_kev=True):
 
         bin_edges = self.bin_edges_kev if use_kev else self.bin_edges_raw
         bin_widths = self.bin_widths_kev if use_kev else self.bin_widths_raw
+        x = np.array(x)
```

Comment:
Maybe `np.asarray` would be a little clearer and we do not need a separate copy of `x`.